utils/convert_gene_names.R

```diff
@@ -0,0 +1,52 @@
+convert_gene_names <- function(genex_df,
+                               gene_column_before,
+                               gene_column_after,
+                               gene_map_df,
+                               map_from,
+                               map_to) {
```

Comment:
```suggestion
convert_gene_names <- function(genex_df,
                               gene_column,
                               gene_map_df,
                               map_from,
                               map_to) {
```
I personally would simplify this so that you don't have so many similar arguments. So to do this I would have just the `gene_column` that contains the IDs to map. Then the function would automatically add a new column that you've set, which always seems to be `gene` based on your current code. 

Along those same lines though, I would actually name the output a less ambiguous name, like maybe the same name as the `map_to` argument. So if you are outputting ensembl id's, then the new column is named `ENSEMBL` or similar, but something that indicates what kind of gene id is present.

Reply:
This is a great suggestion and can help the repo by making this function simpler to use. However I think making this change here would break other stuff, so I need to create an issue to come back to reexamine this later:

https://github.com/AlexsLemonade/medulloblastoma-classifier/issues/62